vpr/test/test_vpr.cpp

```diff
@@ -126,6 +126,8 @@ TEST_CASE("read_rr_graph_metadata", "[vpr]") {
             "wire.eblif",
             "--route_chan_width",
             "100",
+            "--reorder_rr_graph_nodes_algorithm",
+            "random_shuffle"
```

Comment:
This is too early in the flow, as the metadata isn't added until line 147.

Reply:
Fixed. I also tested that commenting out the `std::shuffle` on line 186 causes the test to fail.